addons/account_peppol/models/res_company.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    account_peppol_attachment_ids = fields.Many2many(
+        comodel_name='ir.attachment',
+        string='Peppol Documents',
+    )
+    account_peppol_proxy_state = fields.Selection(
+        selection=[
+            ('not_registered', 'Not registered'),
+            ('pending', 'Pending'),
+            ('manually_approved', 'Approved'),
+            ('active', 'Active'),
+            ('rejected', 'Rejected'),
+        ],
+        string='PEPPOL status',
+        default='not_registered',
+        compute='_compute_account_peppol_proxy_state',
+        readonly=False,
+        store=True,
+    )
+    is_account_peppol_participant = fields.Boolean(string='PEPPOL Participant')
+    peppol_eas = fields.Selection(
+        related='partner_id.peppol_eas',
+        store=True,
```

Comment:
Why store True? (you do not seem to search on it and maybe then you can do partner_id.)

Reply:
I think I assumed this because it was related but it apparently doesn't matter

Reply:
Never mind, it does matter, I tested :laughing: